### PR TITLE
conditional statements/it-academy.by/M-JC1-66-20/German Andrey/ Done tasks 1-14

### PR DESCRIPTION
**Pull request by Germanand196**
## Pull request state

Please indicate actual status for PR:
- [x] on review
- [x] reviewed
- [ ] fixed
- [ ] on approve
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task01.java [hunk 1]
```diff
@@ -40,6 +40,6 @@ public static void main(String[] args) {
     static boolean isTopPlayer(int sizeTop, int positionPlayerByDamage) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return false;
+        return sizeTop>=positionPlayerByDamage;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task01.java:43]: Violations of the Code Convention!

**Review by rakovets — CHANGES_REQUESTED** [review]
The copy is correct.

Now you need to write your solution.

File: src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task01.java (hunk 1 above)
Comment (Germanand196) [reply src/main/java/com/rakovets/course/javabasics/practice/conditionalstatements/Task01.java:43]: fix: tasks 1 for theme 'conditional-statements'

**Comment by Germanand196** [comment]
fix: tasks 1 for theme 'conditional-statements'

**Comment by rakovets** [comment]
> The copy is correct.
> 
> Now you need to write your solution.

You need to write your solution. (not COPY-PASTE)

**Comment by Germanand196** [comment]
fix: tasks 1 - 14 for theme 'conditional-statements'

**Comment by Germanand196** [comment]
fix: tasks 1 - 14 for theme 'conditional-statements'

**Review by rakovets — CHANGES_REQUESTED** [review]
27/165 tests are failed.

**Pull request closed without merging**